packages/core/cli.ts

```diff
@@ -138,8 +138,9 @@ yargs(process.argv.slice(2))
 		},
 		// eslint-disable-next-line @typescript-eslint/no-explicit-any
 		(args: any) => {
+			const fT = path.join(process.cwd(), args.fuzzTarget);
 			const opts: Options = {
-				fuzzTarget: path.join(process.cwd(), args.fuzzTarget),
+				fuzzTarget: fT.endsWith(".js") ? fT : fT + ".js",
```

Comment:
File extensions could also be `.mjs` or `.cjs` to explicitly specify the module system. It's probably ok to expect `.js` if no extension is present.

Reply:
Added!